card/mod/03_machines/lib/card/machine_input.rb

```diff
@@ -2,34 +2,34 @@ class Card
   module MachineInput
     module ClassMethods
       attr_accessor :machines_wql
-    
+
       def machine_input_for args
         @machines_wql = args
       end
-    
+
       def machine_input &block
         define_method :machine_input, block
       end
     end
-  
+
     def self.included host_class
       host_class.extend( ClassMethods )
       host_class.machines_wql = {}
       host_class.machine_input do
         format._render_raw
       end
-      host_class.event "after_machine_input_updated_#{host_class.name.gsub(':','_')}".to_sym, :after=>:extend, :on => :save do
-        machines = Card.search( {:right_plus => [{:codename => "machine_input"}, {:link_to => name}]}.merge(host_class.machines_wql) )  
+      host_class.event "after_machine_input_updated_#{host_class.name.gsub(':','_')}".to_sym, after: :extend, on: :save do
+        machines = Card.search( {right_plus: [{codename: "machine_input"}, {link_to: name}]}.merge(host_class.machines_wql) )
         machines.each do |item|
           item.reset_machine_output! if item.kind_of? Machine
         end
       end
-      
-      host_class.event "before_machine_input_deleted_#{host_class.name.gsub(':','_')}".to_sym, :after=>:approve, :on => :delete do
-        @involved_machines = Card.search( {:right_plus => [{:codename => "machine_input"}, {:link_to => name}]}.merge(host_class.machines_wql) )  
+
+      host_class.event "before_machine_input_deleted_#{host_class.name.gsub(':','_')}".to_sym, after: :approve, on: :delete do
+        @involved_machines = Card.search( {right_plus: [{codename: "machine_input"}, {link_to: name}]}.merge(host_class.machines_wql) )
```

Comment:
Line is too long. [135/80]<br>Space inside { missing.<br>Space inside } missing.<br>Space inside parentheses detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.